### ISSUE TITLE
Behaviour of Enter key - changing the style (Trac #47)

### ISSUE THREAD

**Issue by sleemanj** [sleemanj/xinha#47]
It would be nice (though it is not very high on my priority list) to have this feature:

Let's say you have four styles: h1 (Title), h2 (Subtitle), p (Text) and div.quotation (Quotation). The editor should start in 'Title' mode. Then when user presses Enter the editor should automatically switch to Subtitle (of course, the user could change the mode to some other).
When the user is inside Subtitle the enter key should go to Text. Inside Text mode the enter should keep it there, but it should close the current paragraph and start new one ('</p><p>').

Basically, the developer should have the way to determine starting style and for each style what its 'next style' should be. 

_Reported by anzenews, migrated from http://trac.xinha.org/ticket/47_
